src/components/table/table.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Table, TableHead, TableBody, TableRow, TableCell } from './index';
+import { createWrapper } from '../../test-utils';
+
+function render() {
+  return createWrapper(
+    <Table>
+      <TableHead>
+        <TableCell>id</TableCell>
+        <TableCell>name</TableCell>
+      </TableHead>
+      <TableBody>
+        <TableRow>
+          <TableCell>1</TableCell>
+          <TableCell>Bob Ross</TableCell>
+        </TableRow>
+        <TableRow>
+          <TableCell>2</TableCell>
+          <TableCell>Bob Marley</TableCell>
+        </TableRow>
+      </TableBody>
+    </Table>,
+  );
+}
+
+test('rendering', () => {
+  const wrapper = render();
+  expect(wrapper).toMatchSnapshot();
```

Comment:
Went with the snapshot test here, because this produces a lot of DOM and I didn't want to type out a ton of assertions for each thing. The snapshot test gets us comprehensive tests for all of that, with the downside that it's kind of "out of sight, out of mind", and the developer intent isn't super clear from the test itself.